### ISSUE TITLE
Segmentation Fault

### ISSUE THREAD

**Issue by shawnjhenry** [clementfarabet/lua---nnx#37]
Hi,

I'm attempting to train a network consisting of a recurrent layer on top of a time delay conv net using nn.Recurrent.  It runs fine, albeit slowly, on my Macbook, but when I try to run it on my much faster Ubuntu desktop I get a segmentation fault error with a core dump message.  Where does torch dump the error message?  

In case you want to try reproducing the error, the network looks like this:

memorySize = 10
windowSize = 5
nSamples = 2693
nFeatures = 100
nClasses = 7

TDCNN = nn.Sequential()
TDCNN:add(nn.LookupTable(nSamples, nFeatures))
TDCNN:add(nn.Padding(1, windowSize-1))
TDCNN:add(nn.Padding(1, -windowSize+1))
TDCNN:add(nn.TemporalConvolution(nFeatures, nFeatures, windowSize))
TDCNN:add(nn.Max(1))

RNN = nn.Recurrent(nn.Identity(), TDCNN, nn.Linear(nFeatures, nFeatures), nn.Tanh(), memorySize)

model = nn.Sequential()
model:add(RNN)
model:add(nn.Linear(nFeatures, nClasses))
model:add(nn.LogSoftMax())

loss = nn.ClassNLLCriterion()

Inputs to the network are torch.LongTensors of varying length. 

Thanks,
Shawn

**Comment by nicholas-leonard** [comment]
Hi, the nn.Recurrent in this repository is deprecated. Use the [rnn](https://github.com/Element-Research/rnn) repository instead. For examples, you can checkout [dp](https://github.com/nicholas-leonard/dp/blob/master/examples/recurrentlanguagemodel.lua). If you still have issues, let me know.
